evap/results/views.py

```diff
@@ -86,8 +86,13 @@ def course_detail(request, semester_id, course_id):
                 answers = [answer for answer in result.answers if user_can_see_text_answer(request.user, represented_users, answer, public_view)]
                 if answers:
                     results.append(TextResult(question=result.question, answers=answers))
-            else:
+            elif show_grades or isinstance(result, HeadingResult):
                 results.append(result)
```

Comment:
so why again do we show text answers but not rating answers if not show_grades?

...or rather why do we show a user the results page at all if they cannot see the results?

...can_user_see_results might need a rename because right now they can see results, even though only textanswers... `can_user_see_grades`?

Reply:
text answers are qualitative feedback, the rating answers are quantitative, so we decided not to have thresholds for publishing text answers.

the results page is shown to contributors because there might be text answers. we could also add additional checks and only show it if there really are text answers for the requesting user, but i wouldn't say that's necessary.

renaming to `can_user_see_grades`